### ISSUE TITLE
Plugin works locally with netlify build, but not on app.netlify.com

### ISSUE THREAD

**Issue by moop-moop** [pizzafox/netlify-cache-nextjs#50]
I am doing some initial proof of concepts with Next.js on Netlify.

With the exact same repo, this plugin works fine with the netlify build using netlify-cli locally.

On app.netlify.com, there is no sign the plugin did anything success or error.

[Repo](https://github.com/UWHealth/react-contentful-nextjs-tutorial/tree/becoming-ungatsby)

[Netlify log on app.netlify.com](https://app.netlify.com/sites/stoic-keller-27a01a/deploys/5e90130bbf62a21c9ae07fd7)

Local log Attached.
[ungatsby.log](https://github.com/pizzafox/netlify-cache-nextjs/files/4460244/ungatsby.log)

**Comment by pizzafox** [comment]
Have you enabled the Netlify build beta for that repo? You have to opt-in.

**Comment by alexshyba** [comment]
Hey @moop-moop, I've looked into the repo, everything looks right to me. We got something similar working here, just in case for the reference: https://github.com/uniformdev/sitecore-jss-nextjs-starterkit

As @pizzafox says, once you enable Netlify Build Beta, and build (I had to clear build cache to have it kick off), it should be picking up the new build system and the plugin along the way:
https://app.netlify.com/teams/<your-team>/build-plugins-beta

Another thing to try is to re-create the site in Netlify and enable Build Beta on it immediately after, if it is not picking it up.

This plugin works great for us.
Good luck!
Alex
